saleor/graphql/product/dataloaders/products.py

```diff
@@ -665,6 +665,8 @@ def batch_load(self, keys: Iterable[Tuple[int, int, Optional[str]]]):
         ] = defaultdict()
         for thumbnail in thumbnails:
             format = thumbnail.format.lower() if thumbnail.format else None
+            if format == "original":
```

Comment:
Shouldn't we use the value from `ThumbnailFormat`? I mean `ThumbnailFormat.ORIGINAL` instead of `original`?

Reply:
We could, but we'd need to refactor this function to accept an enum instead of a string.

Reply:
Ah, it's not an enum, it's a class with string attributes, updating code to reflect that.